.azure/pipelines/ci.yml

```diff
@@ -314,7 +314,7 @@ stages:
         displayName: Build Arm64 Installers
 
       # A few files must also go to the VS package feed.
-      - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+      - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), ne(variables['PostBuildSign'], 'true')) }}:
```

Comment:
What sets the `PostBuildSign` variable and the corresponding property in `msbuild`❔ Similarly what sets the `$(DisableMicroBuild)` property in `msbuild`❔

I'm betting you're using these to test things out but am curious&hellip;

Reply:
Right now it's a global variable in the build pipeline. It'll eventually just make its way into the YAML as a variable.

DisableMicrobuild is not set AFAIK